htmd/parameterization/dihedral.py

```diff
@@ -410,6 +410,14 @@ def plotDihedralEnergies(self, idihed):
         fitted_energy = self._fitted_energies[idihed] - np.min(self._fitted_energies[idihed])
         indices = np.argsort(angle)
 
+        path =  os.path.join(self.result_directory, self._names[idihed])
```

Comment:
I would like to keep the `.svg` inside the join function

Reply:
Why?

Reply:
for consistency purposes. adding `.svg` after path building through join may create erroneous strings